index.html

```diff
@@ -450,982 +450,33 @@ <h2>Profiling Mechanism</h2>
   <!-- Core Profile -->
   <section id="core-profile">
     <h2>WoT Core Profile</h2>
```

Comment:
This kind of huge PR cannot be discussed and reviewed in reasonable time.
The Data Model section will be significantly reworked in https://github.com/w3c/wot-profile/pull/93, so this MR should not be merged.

The proposed new text about discovery and link relation types should be carried over to individual MRs to enable timely reviews.

Reply:
@mlagally wrote:
> This kind of huge PR cannot be discussed and reviewed in reasonable time.

There are only 25 lines of new text in this PR, which just provides a list summarising the proposed updated contents of the Core Profile.

The rest is removing what appears to be mostly un-reviewed text which I think should be removed and then re-added piece by piece with the proper level of scrutiny. I note that you have not responded to my proposal in https://github.com/w3c/wot-profile/pull/125#issuecomment-977978088.

> The Data Model section will be significantly reworked in #93, so this MR should not be merged.

@egekorkan and I have already commented extensively on #93 to explain why it doesn't address our concerns. 

> The proposed new text about discovery and link relation types should be carried over to individual MRs to enable timely reviews.

The discovery section has already been split out into a separate PR (#121).

There is an ongoing discussion about a potential links section in #114.

There is just a single sentence about each of those topics in this PR, as part of the list of contents, which is intended to show which parts from the of the Core Profile I propose to keep. I am happy to remove those sentences if those sections don't end up being created, but that doesn't seem to be the biggest issue here.